src/Controller/Speaker/Create.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace App\Controller\Speaker;
+
+use App\Entity\Speaker;
+use App\Form\SpeakerType;
+use App\Repository\SpeakerRepository;
+use App\Service\FileUploader;
+use Symfony\Component\Form\FormFactoryInterface;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\RouterInterface;
+use Twig_Environment;
+
+final class Create
+{
+    private $renderer;
+    private $speakerRepository;
+    private $formFactory;
+    private $router;
+    private $fileUploader;
+
+    public function __construct(
+        Twig_Environment $renderer,
+        SpeakerRepository $speakerRepository,
+        FormFactoryInterface $formFactory,
+        RouterInterface $router,
+        FileUploader $fileUploader
+    )
+    {
+        $this->renderer = $renderer;
+        $this->speakerRepository = $speakerRepository;
+        $this->formFactory = $formFactory;
+        $this->router = $router;
+        $this->fileUploader = $fileUploader;
+    }
+
+    public function handle(Request $request): Response
+    {
+        $speaker = new Speaker();
```

Comment:
Is a speaker valid with no information at all? Should you use a DTO and then use the DTO in a service to hydrate a proper Speaker entity?